Sources/ResourceLocation/ResourceLocation.swift

```diff
@@ -116,8 +116,17 @@ public enum ResourceLocation: Hashable, CustomStringConvertible, Codable {
         switch self {
         case .localFilePath(let path):
             return path
-        case .remoteUrl(let url, _):
-            return url.absoluteString
+        case .remoteUrl(let url, let headers):
+            guard let headers = headers,
+                  headers.count > 0 else {
+                return url.absoluteString
+            }
+            let params: [String: Encodable] = [
+                "url": url.absoluteString,
+                "headers": headers
+            ]
+            let data = try! JSONSerialization.data(withJSONObject: params, options: .fragmentsAllowed)
+            return String(data: data, encoding: .utf8) ?? ""
```

Comment:
`try!` may cause a potential crash. Could you please make it `func stringValue() throws -> String` instead.

I've checked, there are not too many places where this string is used. Should be fairly straightforward to change property to a throwing method call.

Reply:
change var to throwable func